Formula/tpp.rb

```diff
@@ -18,6 +18,8 @@ class Tpp < Formula
     sha256 cellar: :any_skip_relocation, el_capitan:  "25e92e9f229433131cc82cf48a3cec90d19a28a08a56fadcc095b1ecf4df2304"
   end
 
+  deprecate! date: "2022-04-14", because: :repo_archived
```

Comment:
We don't list a repo anywhere in this formula, so probably better to go for `:unmaintained`.

(Commit message also isn't quite right now).

Reply:
This should be ready to go now.